static/src/javascripts/projects/commercial/modules/hosted/onward.js

```diff
@@ -1,54 +1,61 @@
 // @flow
 import config from 'lib/config';
-import mediator from 'lib/mediator';
 import fetchJson from 'lib/fetch-json';
 import fastdom from 'lib/fastdom-promise';
-import HostedCarousel from 'commercial/modules/hosted/onward-journey-carousel';
+import { initHostedCarousel } from './onward-journey-carousel';
 
-const loadOnwardComponent = (
+const getPlaceholdersFromDom = (dom = document): Array<any> =>
+    [].slice.call(dom.getElementsByClassName('js-onward-placeholder'));
```

Comment:
I prefer the notation: `[...dom.getElementsByClassName('js-onward-placeholder')]` or `Array.from(dom.getElementsByClassName('js-onward-placeholder'))`.